l10n/en/careers/404.ftl

```diff
@@ -0,0 +1,10 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/careers/404/
+
+job-not-found-page-not-found-page-page-not-found = 404: Job Not Found
+job-not-found-page-sorry-we-cant-find-that-job = Sorry, we can't find that job posting
+job-not-found-page-not-open-position = The link you followed doesn't seem to for be an open position.
```

Comment:
Nit: grammar and punctuation
```suggestion
job-not-found-page-not-open-position = The link you followed doesn’t seem to be an open position.
```

Reply:
Ugh @craigcook 🤦 - serves me right for late-evening coding and PRing rather than sleeping on it first!